docs/settings.md

```diff
@@ -63,6 +63,7 @@ connecting IPs in the `forwarded-allow-ips` configuration.
 ## HTTPS
 
 * `--ssl-keyfile <path>` - SSL key file
+* `--ssl-keyfile-password <str>` - SSL key file password
```

Comment:
Good catch, could we just add that this is in case the key is encrypted ?
Something like this?
```suggestion
* `--ssl-keyfile-password <str>` - SSL key file password for encrypted keys
```

Reply:
hi @euri10, what do you think of  "Password to decrypt ssl key" ?

Reply:
lgtm 😎